packages/oauth/oauth-provider/src/access-token/access-token-mode.ts

```diff
@@ -0,0 +1,4 @@
+export enum AccessTokenMode {
```

Comment:
All tokens are now wrapped inside JWTs. The reason for this is that before, the `TokenId` string was used directly as access-token. However, these `TokenId` are now exposed through the account management API.

In order to avoid exposing valid access-tokens to the user, the `TokenId` are now useless unless wrapped inside a JWT.